stripe-core/src/main/java/com/stripe/android/core/networking/RequestHeadersFactory.kt

```diff
@@ -35,6 +37,16 @@ sealed class RequestHeadersFactory {
 
     protected open var postHeaders: Map<String, String> = emptyMap()
 
+    protected abstract val xStripeUserAgent: String
+
+    protected fun defaultXStripeUserAgentMap() = mutableMapOf<String, String?>(
+        LANG to KOTLIN,
+        AnalyticsFields.BINDINGS_VERSION to StripeSdkVersion.VERSION_NAME,
+        AnalyticsFields.OS_VERSION to "${Build.VERSION.SDK_INT}",
+        TYPE to "${Build.MANUFACTURER}_${Build.BRAND}_${Build.MODEL}",
```

Comment:
Should we remove model from here since it's already in `MODEL` below?

Reply:
Some other places are setting `AnalyticsFields.DEVICE_TYPE` to "${Build.MANUFACTURER}_${Build.BRAND}_${Build.MODEL}", I think it's better to keep in sync